src/components/utils.js

```diff
@@ -71,3 +71,15 @@ export function fromEntries(iterable) {
   return [...iterable]
     .reduce((obj, { 0: key, 1: val }) => Object.assign(obj, { [key]: val }), {});
 }
+
+/**
+ * Get the DOM node for the .vitessce-container element.
+ * @returns {Element} The vitessce-container DOM node or document.body.
+ */
+export function getVitessceContainer() {
+  const els = document.getElementsByClassName('vitessce-container');
+  if (els.length === 1) {
+    return els[0];
+  }
+  return document.body;
+}
```

Comment:
I'm not sure how this is used, but I'm pretty sure Vitessce should never be messing with the document root. I don't want #459 to be undone. Sorry if I'm not getting it.

Reply:
That makes sense I will think about other approaches to use here.

The issue is that the Popover element by default is rendered as a child of `document.body` (see `getPopupContainer` description [here](https://ant.design/components/tooltip/#API)) but the theme styles are applied by adding a theme class to the `.vitessce-container` div. So when the popover element is outside that div, the theme cannot be accessed. 

A different approach I can try is to render an empty div element as a sibling of the Popover component, use a React `ref` to get the DOM node object for the empty div, and then pass that DOM node to `getPopupContainer`

Reply:
<img width="933" alt="Screen Shot 2020-03-20 at 11 39 21 PM" src="https://user-images.githubusercontent.com/7525285/77218758-1b954f80-6b05-11ea-8c02-459d839f67b7.png">

Reply:
Cool: Either way, I think it might be a little confusing to someone down the road, so may just add the explanation above in a comment, and use whichever approach seems best to you. I'm not sure myself. Thanks!